composer.json

```diff
@@ -39,6 +39,7 @@
         "symfony/twig-bridge": "^3.4",
         "symfony/twig-bundle": "^3.4",
         "symfony/validator": "^3.4",
+        "symfony/web-server-bundle": "^3.4",
```

Comment:
Is there a way to only require this as a dev-dependency?

Reply:
Done, moved to require-dev